components/prism-avisynth.js

```diff
@@ -0,0 +1,169 @@
+			pixel: 'blur sharpen generalconvolution (?:spatial|temporal)soften fixbrokenchromaupsampling',
+			timeline: 'trim (?:un)?alignedsplice (?:assume|assumescaled|change|convert)FPS (?:delete|duplicate)frame dissolve fade(?:in|out|io)[02]? freezeframe interleave loop reverse select(?:even|odd|(?:range)?every)',
+			interlace: 'assume(?:frame|field)based assume[bt]ff bob complementparity doubleweave peculiarblend pulldown separate(?:columns|rows|fields) swapfields weave(?:columns|rows)?',
+			audio: 'amplify(?:db)? assumesamplerate audiodub(?:ex)? audiotrim convertaudioto(?:(?:8|16|24|32)bit|float) converttomono delayaudio ensurevbrmp3sync get(?:left|right)?channel kill(?:audio|video) mergechannels mixaudio monotostereo normalize resampleaudio supereq ssrc timestretch',
+			conditional: 'conditional(?:filter|select|reader) frameevaluate scriptclip writefile(?:if|start|end)? animate applyrange tcp(?:server|source)',
+			export: 'imagewriter',
+			debug: 'subtitle blankclip blackness colorbars(?:hd)? compare dumpfiltergraph setgraphanalysis echo histogram info messageclip preroll showfiveversions show(?:framenumber|smpte|time) stack(?:horizontal|vertical) tone version'
+		}
+	};
+
+	var types = toAlternation([wordLists.types]);
+	var keywords = toAlternation([wordLists.keywords]);
+	var predefined = toAlternation([wordLists.predefined]);
+	var constants = toAlternation([wordLists.constants]);
+	var properties = toAlternation(Object.values(wordLists.properties));
+	var intfuncs = toAlternation(Object.values(wordLists.internals));
+	var intfilters = toAlternation(Object.values(wordLists.filters));
+	var internals = cat(intfuncs, cat(properties, intfilters));
+
+	Prism.languages.avisynth = {
+
+		'comment': [
+			{	// Matches [* *] nestable block comments, but can not handle nested comments correctly (recursion)
+				pattern: /(^|[^\\])\[\*[\s\S]*?(?:\*\]|$)/,
+				lookbehind: true,
+				greedy: true
+			},
+			{	// Matches /* */ block comments
+				pattern: /(^|[^\\])\/\*[\s\S]*?(?:\*\/|$)/,
+				lookbehind: true,
+				greedy: true
+			},
+			{	// Matches # comments
+				pattern: /(^|[^\\$])#.*/,
+				lookbehind: true,
+				greedy: true
+			}
+		],
+
+		// Handle before strings because optional arguments are surrounded by double quotes.
+		'argument': {
+			pattern: re(/\b(?:<<0>>)\s+("?)\w+\1/.source, [types], 'i'),
+			inside: {
+				'keyword': {
+					pattern: re(/\b(?:<<0>>)\b/.source, [types], 'i')
+				}
+			}
+		},
+
+		'string': [
+			{ // triple double-quoted
+				pattern: /"""[\s\S]*?"""/i,
+				greedy: true,
+			},
+			{ // single double-quoted
+				pattern: /"(?:\\(?:\r\n|[\s\S])|(?!")[^\\\r\n])*"/,
+				greedy: true,
+				inside: {
+					'constant': {
+						pattern: re(/<<0>>/.source, [predefined]) // These *are* case-sensitive!
+					}
+				}
+			}
+		],
+
+		// The special "last" variable that takes the value of the last implicitly returned clip.
+		'variable': /\b(last)\b/i,
```

Comment:
This is why the tests fail.

```suggestion
		'variable': /\b(?:last)\b/i,
```

Reply:
Guess who ran the language tests but never ran the rest of the test suite to catch stuff like this.